docs/src/pages/components/radio-buttons/ErrorRadios.tsx

```diff
@@ -47,6 +47,7 @@ export default function ErrorRadios() {
   return (
     <form onSubmit={handleSubmit}>
       <FormControl
+        variant="standard"
```

Comment:
Does it have an impact on radios? I would be surprised if it has one

Reply:
Unfortunately, it impacts the `FormHelperText` padding.

Reply:
Ah right, ok makes sense.